tests/plugin_qpp/qpp_client.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <qemu-plugin.h>
+#include <plugin-qpp.h>
+#include <glib.h>
+
+QEMU_PLUGIN_EXPORT const char *qemu_plugin_name = "qpp_client";
+QEMU_PLUGIN_EXPORT int qemu_plugin_version = QEMU_PLUGIN_VERSION;
+QEMU_PLUGIN_EXPORT const char *qemu_plugin_uses[] = {"qpp_srv", NULL};
+
+#include "qpp_srv.h"
+
+void my_cb_exit_callback(gpointer evdata, gpointer udata);
+
+QEMU_PLUGIN_EXPORT void my_cb_exit_callback(gpointer evdata, gpointer udata) {
+    qemu_plugin_outs("called my on exit callback\n");
```

Comment:
Would there be a way to make this test fail if this doesn't get called? Perhaps some kind of global variable gets set in here and then on exit (but somehow after the on_exit cb?) we assert that it's set

Reply:
I can't think of a way to assert after the cb as it should be the last thing to run